roles/maintenance_14_sles/tasks/main.yml

```
@@ -0,0 +1,76 @@
+---
+
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignoreme
+    name: bar
+
+- <<: *task
+  vars:
+    taskid: 14-001
+    name: "Apparmor: Check if apparmor is enabled and enforcing apparmor_status"
+  ansible.builtin.command: aa-status
+  register: apparmor_status_output
+  changed_when: false
+  failed_when: false
+
+- <<: *task
+  vars:
+    taskid: 14-001
+    name: "Apparmor: Check if apparmor is enabled and enforcing apparmor_status"
+  ansible.builtin.debug:
+    msg: "{{ apparmor_status_output.stdout }}"
+  when: apparmor_status_output is succeeded
```

On doing some [additional checking](https://manpages.ubuntu.com/manpages/focal/en/man8/aa-status.8.html), you can simply check the rc of aa-status.
```suggestion
  changed_when: "task.rc != 0"
  failed_when: "task.rc == 4"  # Insufficient permission to check AA policies.
```